api/HTMLMediaElement.json

```diff
@@ -47,6 +47,429 @@
           "deprecated": false
         }
       },
+      "HAVE_CURRENT_DATA": {
```

Comment:
These are constants and should be reverted per discussion in https://github.com/mdn/browser-compat-data/issues/7585.